### ISSUE TITLE
Syncing labels and status field

### ISSUE THREAD

**Issue by BeepBot99** [Mesure73L/mesure.x10.mx#61]
### What problem were you having?

We should not have status-related labels, such as `ready to merge` and `in progress`. Because, there are already statuses in the project.

### What solution would you like?

Remove all the status related labels

### What alternatives have you considered?

Make it a lot harder to automate

#### Pros
* You can still see the status on the side
* It is easier
#### Cons
* IDK

### What steps need to be done?

- [ ] Update status field after labels are updated
- [ ] Update labels after status field is updated

**Comment by BeepBot99** [comment]
Also remove high priority and low priority, those can go in the project too.

**Comment by BeepBot99** [comment]
I will do it all.

**Comment by BeepBot99** [comment]
@Mesure73L Do I have permission to start?

**Comment by BeepBot99** [comment]
OK I started.

**Comment by Mesure73L** [comment]
What? I never said you could start.

**Comment by Mesure73L** [comment]
Please undo whatever you did.

**Comment by Mesure73L** [comment]
## The reason why this isn't accepted:

Not everything is Twelve of Code, therefore not in the project. The labels also allow you to see the status easily while in the issues page. There is a use for those labels, and they should not be deleted.

Please undo everything you did related to this issue, and read #62.

**Comment by BeepBot99** [comment]
## Comments on what you just said
> Not everything is Twelve of Code, therefore not in the project. 

Yes, I know, I realized that after I did it.
> The labels also allow you to see the status easily while in the issues page.

It can be configured so project statuses do too.
> The labels also allow you to filter by priority, so you can see everything that is high priority and not assigned, for example.

You can create a priority field too. In addition, having two separate fields for priority and status can help you sort it even better.

> Please undo everything you did related to this issue, and read #62.

I will not do that (yet), read this first.
## Options from here
1. **This is my top choice**. I know you will probably not do it, but I am just mentioning it to say that it is still an option. It is to create a new repository for 12oC. Then, everything will be seperate
2. Create a project for the main website, in addition to the one for 12oC. Then, both projects can have their own stuff, and there will be no need for labels.
3. Keep it as it is and add the labels back, but use the labels only for non-12oC stuff. If we do this, then there will also need to be seperate issue templates for 12oC.

Tell me which one you choose, and I will do it.

**Comment by BeepBot99** [comment]
@Mesure73L Answer this whenever you are ready to, I will not do anything until then. **Please do not do anything related to the project will me showing you how first. It is not completely set up yet.**

**Comment by BeepBot99** [comment]
Basically, we can't use labels for 12oC. It is just a matter of what you want to do with the rest of the website.

**Comment by Mesure73L** [comment]
@BeepBot99

**Comment by BeepBot99** [comment]
What do you mean without you showing me how first?

**Comment by BeepBot99** [comment]
We cannot use labels because you cannot use them with projects (unless you are willing to pay 10 dollars a month.)

**Comment by BeepBot99** [comment]
@Mesure73L

**Comment by Mesure73L** [comment]
> What do you mean without you showing me how first?

Read it again. (its your message lol)



> We cannot use labels because you cannot use them with projects (unless you are willing to pay 10 dollars a month.)

WHat else would I get with it?

**Comment by Mesure73L** [comment]
Please send me the link to that subscription.

**Comment by BeepBot99** [comment]
https://github.com/pricing
You would need to get team.
You would have to pay 8 dollars a month (two people) because I am not paying for it because I don't care and it doesn't work unless everyone has it.

**Comment by BeepBot99** [comment]
It would be a lot harder to set up.
I can add the statuses back, they just wouldn't be used for 12oC.

**Comment by Mesure73L** [comment]
Why can't we use labels with 12oC? How does it know it's status related

**Comment by BeepBot99** [comment]
Do you want a short explanation or a long one? A long one will take me a few minutes to type.

**Comment by Mesure73L** [comment]
> Do you want a short explanation or a long one? A long one will take me a few minutes to type.

Long one

**Comment by Mesure73L** [comment]
Oh you weren't lying about that the longer one would take more time to type

**Comment by Mesure73L** [comment]
ok this is a lot more than a few minutes

**Comment by Mesure73L** [comment]
@BeepBot99 can you please give the explanation?

**Comment by BeepBot99** [comment]
I'm at school you know, be patient please /srs.
I'll do it now.

**Comment by Mesure73L** [comment]
> be patient please /srs

take your own advice bro

**Comment by BeepBot99** [comment]
So, the problem is basically that they are two seperate things.
It is possible to have both at the same time, there is no problem with that.
However, they will not by synced.
They will be two completely seperate systems.
If there is only one system in place, not only will it be significantly easier to manage, but it will also be easier to see insights into it.
You can pay to sync the two systems.

**Comment by BeepBot99** [comment]
OK. But, the statuses will ALWAYS have top priority NO MATTER WHAT.

**Comment by BeepBot99** [comment]
I will do it later. Only status and priority labels were removed. You can do it now if you want.

**Comment by BeepBot99** [comment]
@Mesure73L Can we call tonight to talk about this?

**Comment by BeepBot99** [comment]
https://github.com/marketplace/actions/project-beta-automations
I'm going to try this in a testing repository.

**Comment by BeepBot99** [comment]
Merging this into #51.

**Comment by BeepBot99** [comment]
Unmerging this.

**Comment by Mesure73L** [comment]
@BeepBot99 Please recreate the labels.

**Comment by BeepBot99** [comment]
I'm going to start doing this, but I might not finish.

**Comment by BeepBot99** [comment]
Hi, currently waiting on [this](https://github.com/leonsteinhaeuser/project-beta-automations/discussions/86).

**Comment by BeepBot99** [comment]
@Mesure73L  It is possible and easy to make it so that when the labels change, the item's status in the project changes. However, that is only one way. Currently, there is no GitHub Actions trigger to detect when an item changes status in a project. The solution to that would be to check every issue's status every hour, and then update the labels only every hour. Is that fine with you.
In addition, what would happen if, for instance, it has two different contradicting labels? It cannot have two different statuses.

**Comment by BeepBot99** [comment]
We probably shouldn't run it every hour though, because we only get 2,000 Actions minutes per month.

**Comment by Mesure73L** [comment]
I think this problem is actually fixed, by creating the other project. We do not need the labels anymore,

**Comment by BeepBot99** [comment]
Okay, good. I had thought that this wasn't completed for two reasons:
1. Originally, I had suggested having two different projects, and you had said no.
2. Once we had two seperate projects, you said I still needed to attempt this when I could finish.

However, it doesn't matter anymore, because it is fixed now!
